bin/rails

```diff
@@ -1,9 +1,4 @@
 #!/usr/bin/env ruby
-begin
```

Comment:
what's happening here?

Reply:
I think this can be deleted. there was a problem locally that needed this reset.